src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,40 @@
 package core.basesyntax;
 
+
+import java.io.*;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
[checklist](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file): Divide program logic
Before you create PR with your homework, you need to go through the checklist under the task, and correct all the points described there. The mentor will not check the work until the checklist points are corrected